src/utils/constants.js

```diff
@@ -23,10 +23,48 @@ export const homeUrl = {
   Teacher: "/app/teachers",
   Principal: "/app/principal",
   Student: "/app/student-home",
+  Admin: "/app/admin",
 };
 
 export const dashboardSideBarLinks = {
-  Admin: [],
+  Admin: [
```

Comment:
Please go through the old admin and check the links there. This does not seem complete